examples/webpack.config.local.js

```diff
@@ -50,7 +55,34 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR) {
         // Imports kepler.gl library from the src directory in this repo
         'kepler.gl': SRC_DIR,
         react: resolve(EXAMPLE_DIR, './node_modules/react'),
-        'styled-components': resolve(EXAMPLE_DIR, './node_modules/styled-components')
+        'styled-components': resolve(
```

Comment:
@heshan0131 I think we should use luma/deck/loaders from a master checkout instead of using node modules.
This way we can develop in luma/deck/loaders and have live reload.

Reply:
sure, that requires luma/deck/loaders be cloned to a local folder. You can change `NODE_MODULES_DIR` to a custom folder that contains all 3 modules manually. Most of the time we just want to make sure the version of luma/deck/loaders leasted in package.json should work properly